src/topology/continuous_function/bounded.lean

```diff
@@ -1030,4 +1031,101 @@ module over the algebra of bounded continuous functions from `α` to `𝕜`. -/
 
 end normed_algebra
 
+section normed_lattice_ordered_group
+
+variables [topological_space α] [normed_lattice_add_comm_group β]
+
+-- Can we infer this from https://github.com/leanprover-community/mathlib/blob/f29b0b49badab1bcbe338cb79d102e36e79dce09/src/topology/continuous_function/basic.lean#L114 ?
+instance : partial_order (α →ᵇ β) := partial_order.lift (λ f, f.to_fun) (by tidy)
+
+/--
+Continuous normed lattice group valued functions form a meet-semilattice
+-/
+instance : semilattice_inf (α →ᵇ β) :=
+{ inf := λf g,
+  { to_fun := λ t, f(t)⊓g(t),
+    continuous_to_fun := begin
+      continuity,
+    end,
+    bounded' := begin
+      cases f.bounded' with C₁ hf,
+      cases g.bounded' with C₂ hg,
+      use C₁+C₂,
+      intros,
+      simp,
+      rw normed_group.dist_eq,
+      apply le_trans (norm_inf_sub_inf_le_add_norm _ _ _ _) _,
+      rw [← normed_group.dist_eq, ← normed_group.dist_eq],
+      apply add_le_add (hf _ _) (hg _ _),
+    end },
+  inf_le_left :=  λ f g, continuous_map.le_def.mpr (begin intro, apply inf_le_left, end),
+  inf_le_right := λ f g, continuous_map.le_def.mpr (begin intro, apply inf_le_right, end),
+  le_inf := λ f g₁ g₂ w₁ w₂, continuous_map.le_def.mpr (λ a, begin
+    apply le_inf (continuous_map.le_def.mp w₁ a) (continuous_map.le_def.mp w₂ a),
```

Comment:
To help readability, please replace any `apply` or `refine` which closes a goal by `exact`.
```suggestion
    exact le_inf (continuous_map.le_def.mp w₁ a) (continuous_map.le_def.mp w₂ a),
```